javatests/arcs/core/crdt/CrdtSetTest.kt

```diff
@@ -144,6 +145,28 @@ class CrdtSetTest {
         ).isFalse()
     }
 
+    @Test
+    fun merge_reportsRemovalsFromMergedModel() {
```

Comment:
Theoretically this is also tested below, but it was useful for me to create a very specific test when tracking this issue down, and I figured I might as well keep it.